app/customer.py

```diff
@@ -0,0 +1,14 @@
+from app.shop import Shop
+from math import dist
```

Comment:
Please be more accurate with imports: built-in import is first, than blank line than you own import